src/site/stages/build/drupal/graphql/healthCareRegionPage.graphql.js

```diff
@@ -136,11 +114,6 @@ module.exports = `
     fieldClinicalHealthCareServi {
       processed
     }
-    ${
-      cmsFeatureFlags.FEATURE_FEATURED_HEALTH_SERVICE_CONTENT
-        ? healthCareRegionFeaturedHealthServices
-        : ''
-    }
```

Comment:
I pulled it out because the build breaks with a "hey this field doesn't exist" message. I assumed the field was no longer in use - do you know if it's in still in play, @jonbot ?